pages/api/longestStreak.ts

```diff
@@ -0,0 +1,126 @@
+import { shuffle } from "lodash";
+import { BlockState } from "../../components/math/longestStreak/MultiplicationBlock";
+import { GameLevel } from "../../redux/longestStreakSlice";
+import { getRndInteger } from "./random";
+
+export type GameBlockState = {
+  text: string;
+  value: number;
+  isProduct: boolean;
+  state: BlockState;
+};
+
+export function getRangeValues(level: GameLevel) {
```

Comment:
this is fine for now, but if you expect to add more properties to a GameLevel you might be better off defining a GameLevel type that has 2 properties: `min` and `max`

then you could replace this function with an object called `gameLevelsMetaData`

```
{
  beginner: {
    min: 1,
    max: 10
  },
  beginnerAdvanced: {
    min: 10,
    max: 20,
  },
  ...
}
```

this way if you want to add more levels, its easy to add to this object. or if you want levels to support more properties, you can easily do that as well. 

You can also leave it as is, since your code seems to work properly already.

Reply:
refactored with an object to manage levels.